cart/application/cartService.go

```diff
@@ -47,12 +47,24 @@ type (
 		RestrictionResult validation.RestrictionResult
 	}
 
+	// AdjustmentResult restriction result enriched with the respective item
+	AdjustmentResult struct {
+		Item              cartDomain.Item
```

Comment:
Item -> rename to OriginalItem ?
Add "NewQty int" ?

Reply:
Renamed to OriginalItem and added new field NewQty.